refinery/file_store/migrations/0003_filetypes_and_fileextensions.py

```diff
@@ -0,0 +1,378 @@
+# encoding: utf8
+
+from django.db import migrations, connection
+from django.db.models import Max
+
+def forwards(apps, schema_editor):
+    Filetype = apps.get_model("file_store", "Filetype")
+    FileExtension = apps.get_model("file_store", "FileExtension")
+
+    if Filetype.objects.count():
```

Comment:
This issue doesn't affect "clean" instances without any prior FileType/Extension information, so theres no need to reset the postgres id sequences